packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -269,7 +269,7 @@ const AccountSwitcher = props => {
 
     const findServerForAccount = acc => {
         const server_name = acc.error ? acc.error.details.server : acc.server;
-        return props.mt5_login_list.length > 1
+        return props.mt5_login_list.length >= 1
```

Comment:
Having 1 option this function return null instead of this option. AccountDisplay gets null during displaying options in account switcher